randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+    "2077227745": [
+        # Meta Ridley Boost Tip
+        # Replaces Meta Ridley scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Meta Ridley",
+        (
+            "&main-color=#89D6FF;During the first phase of combat against Meta Ridley, "
+            "hitting it with a &main-color=#FF3333;Boost Ball&main-color=#89D6FF; "
+            "charge causes enough damage to immediately move to the fight's second phase."
+        ),
+    ],
+    "2967215000": [
+        # Non-Superheated Back Magmoor Tip
+        # Replaces Puddle Spore scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Puddle Spore",
+        (
+            "&main-color=#89D6FF;Everything south of Twin Fires Tunnel in Magmoor Caverns is not superheated,\n\n"
+            "allowing for safe travel without the Varia Suit."
+        ),
+    ],
+    "1735564172": [
+        # Arboretum Sandstone PBs Tip
+        # Replaces Arboretum sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;The &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures in this room are "
+            "some of the few that cannot be destroyed by Power Bombs.\nNote that some game versions change that."
+        ),
+    ],
+    "1618265307": [
+        # Burn Dome PB Sandstone Tip
+        # Replaces Burn Dome sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "745991660": [
+        # Gathering Hall PB talloric alloy Tip
+        # Replaces Gathering Hall lamp scans
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Talloric Alloy&main-color=#89D6FF; structures, "
+            "the grate above can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "3344095433": [
+        # Ruined Shrine PB Sandstone Tip
+        # Replaces Ruined Shrine sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "3485565452": [
+        # Vault PB Slots Tip
+        # Replaces Vault bomb slot seal scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;These Bomb Slot covers are able to be destroyed by &main-color=#FF3333;Power Bombs"
+            "&main-color=#89D6FF;.\n\nAdditionally, these Bomb Slots are also able to be activated by Power Bombs."
```

Comment:
`...can be destroyed`/`...can also be activated` flows a bit better than `...are able to be destroyed`/`...are also able to be activated`.